open_optical_gating/cli/determine_reference_period.py

```diff
@@ -100,8 +134,11 @@ def calculate_period_length(diffs):
     return diffs.size - interpolatedMatchEntry
 
 
-def estimate_period_length(diffs):
-    # unlike JTs codes I assume 1 period with a start frame of 1
+def estimate_integer_period_length(diffs):
+    # Unlike JTs codes, this function currently only supports determining the period for a *one* beat sequence.
+    # It therefore also only supports determining a period which ends with the final frame in the diffs sequence.
+    # TODO: JT writes: this function needs a proper function header comment, but first we need to think about whether there's any point having
+    # this as a separate function. Is there any purpose to separating this out from calculate_period_length?
```

Comment:
Three year old comment: I would guess that I separated it because it was separated in your ObjC codes at the time?

Reply:
(And, no, there's no reason for it to be separate)